pkg/orchestrator/pod.go

```diff
@@ -127,9 +127,25 @@ func ScrubContainer(c *v1.Container, scrubber *DataScrubber) {
 		}
 	}()
 
-	// scrub command line
-	scrubbedCmd, _ := scrubber.ScrubSimpleCommand(c.Command)
-	c.Command = scrubbedCmd
+	// scrub args and commands
+	merged := append(c.Command, c.Args...)
+	words := 0
+	for _, cmd := range c.Command {
+		words += len(strings.Split(cmd, " "))
```

Comment:
Could a custom regex / sensitive pattern "replace two words into one"  one day and break this?